guide/additional-info/changes-in-v12.md

```diff
@@ -1977,7 +1977,7 @@ Both the `receiver.createOpusStream()` and `receiver.createPCMStream()` methods
 
 #### VoiceReceiver#destroy
 
-This method has been removed entirely, refer to `StreamDispatcher#destroy` for documentation.
+This method has been removed entirely; refer to `StreamDispatcher#destroy` for documentation.
```

Comment:
The DJS docs don't have docs to `StreamDispatcher#destroy` since it extends [`stream.Writable`](https://nodejs.org/api/stream.html#stream_class_stream_writable), so do you want me to link the docs and rename it to`WritableStream#destroy`, link but don't rename it, or keep it as is?